### PR TITLE
ActionBar now supports all versions

### PR DESCRIPTION
**Pull request by montlikadani**

### CODE REVIEW

**Comment by tastybento** [comment]
Have you managed to test it on all the server versions?

**Comment by montlikadani** [comment]
It works well and I use it too.

File: src/com/wasteofplastic/askyblock/events/ActionBarMessageEvent.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
+package hcom.wasteofplastic.askyblock.events;
```
Comment (Phoenix616) [inline src/com/wasteofplastic/askyblock/events/ActionBarMessageEvent.java]: This is the wrong package name...

File: src/com/wasteofplastic/askyblock/ActionBar.java [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+	public static String nmsver;
+	public static boolean works = true;
+	static boolean useOldMethods = false;
+	public static Plugin plugin;
+
+	public static void sendActionBar(Player player, String message) {
+		if (!player.isOnline()) {
+			return;
+		}
+		ActionBarMessageEvent actionBarMessageEvent = new ActionBarMessageEvent(player, message);
```
Comment (Phoenix616) [inline src/com/wasteofplastic/askyblock/ActionBar.java]: Import is missing.

**Comment by tastybento** [comment]
Yeah, I'm seeing NPE's. I'm going to pull this back out. I'm not happy with including NMS stuff anyway. Sorry.

**Comment by montlikadani** [comment]
Ooops... Sorry. You should not have merged it because I would have fixed it.

**Comment by tastybento** [comment]
Thanks for the push, but I'm really trying to avoid adding any more features to ASkyBlock because of the support load it'll incur.

**Pull request merged**